controller/pagecontroller.php

```diff
@@ -53,6 +53,13 @@ public function index() {
 			$csp->addAllowedImageDomain('https://api.tiles.mapbox.com');
 			// inline images
 			$csp->addAllowedScriptDomain('data:');
+			// nominatim geocoder
+			$csp->addAllowedScriptDomain('http://nominatim.openstreetmap.org/search?q=*');
+			$csp->addAllowedScriptDomain('http://nominatim.openstreetmap.org/reverse');
+			$csp->addAllowedConnectDomain('http://router.project-osrm.org');
+			// mapzen geocoder
+			$csp->addAllowedConnectDomain('http://search.mapzen.com/v1/search?');
```

Comment:
Would make sense. That said, the rules are already pretty dubious due to line 55. You don't allow all inline images here but allow arbitrary script execution within `data:`. If you really need this then that would not provide that much additional security as it basically disables the CSP protection totally. Maybe you meant `addAllowedImageDomain`?